### PR TITLE
Update connecting-to-mongodb.rst

### PR DESCRIPTION
**Pull request by orisdorch**
Added instruction for previously configured nodes. Elaborated on risk of opening the db port.

### CODE REVIEW

File: source/guides/network/connecting-to-mongodb.rst [hunk 1]
```diff
@@ -42,7 +42,14 @@ And then use this file when configuring ``symbol-bootstrap``, for example:
 
     symbol-bootstrap start -p testnet -a dual -c custom_parameters.yml
 
-This will make the database's TCP port ``27017`` accessible from the host so be careful, and only use this feature **for development purposes**.
+Previously configured nodes may need to be stopped and restarted with the --upgrade -d argument in order to process the updated custom yml file:
```
Comment (segfaultxavi) [inline source/guides/network/connecting-to-mongodb.rst:45]:
```suggestion
Previously configured nodes may need to be stopped and restarted with the ``--upgrade -d`` argument in order to process the updated custom yml file:
```

Things that the reader should copy verbatim, like file names or arguments, should be between double backticks.

Also, when used for the first time, I like to use the long version of parameters. Right now I don't remember what ``-d`` does! :D

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

File: source/guides/network/connecting-to-mongodb.rst (hunk 1 above)
Comment (orisdorch) [reply source/guides/network/connecting-to-mongodb.rst]: Oh drats, may have messed up with the review - still getting familiar with Github.. We can remove -d entirely, but should ALSO consider recommending that the user backs up their target folder before using the --upgrade command.

`  --upgrade
      It regenerates the configuration reusing the previous keys. Use this flag when upgrading the version of bootstrap to
      keep your node up to date without dropping the local data. **Backup the target folder before upgrading.**`

**Review by segfaultxavi — DISMISSED** [review]
Looks good to me!

If you want to walk the extra mile (and learn a bit of RST in the process), why don't you make the last sentence a WARNING box?
It's something like:
```rst
.. caution:: This will make the database's TCP port...
```

Otherwise, it's OK to land, but I think somebody from the organization should do it. @0x6861746366574 ?

**Comment by segfaultxavi** [comment]
Note: Please use "Squash and merge".

**Comment by orisdorch** [comment]
Added the proposed admonition - apologies for the slew of commits! Definitely prefer "Squash and merge".

**Review by segfaultxavi — APPROVED** [review]
Nothing wrong with the slew of commits, that's definitely the Open Source Way 😄

**Pull request merged**